AfroLayout.podspec

```diff
@@ -8,7 +8,7 @@
 
 Pod::Spec.new do |s|
   s.name             = "AfroLayout"
-  s.version          = "0.1.6"
+  s.version          = "3.0.0"
```

Comment:
Yeah, make it at `0.2` because it is still a beta pod. I will tag after the merge